flink-connector-oceanbase-cdc/src/test/resources/ddl/column_type_test.sql

```diff
@@ -40,11 +43,12 @@ CREATE TABLE full_types
     big_decimal_c DECIMAL(65, 1),
     date_c        DATE,
     time_c        TIME(0),
-    datetime3_c   TIMESTAMP(3),
-    datetime6_c   TIMESTAMP(6),
```

Comment:
We also need to add a column whose type is `TIMESTAMP(p)`.

Reply:
OK.